packages/core/interfaces/src/compilerSupport/compilerSupportInterface.ts

```diff
@@ -42,7 +41,7 @@ export interface CallAquaFunctionArgs {
   /**
    * Peer to call the function on
    */
-  peer: IFluenceInternalApi;
+  peer: unknown;
```

Comment:
that's a little weird

Reply:
This should be refactored and moved to JS-client. Previous type was essentially the same. I will add `TODO` to remember this.